core/controllers/email_dashboard.py

```diff
@@ -235,9 +235,9 @@ def post(self, query_id):
             raise self.UnauthorizedUserException(
                 '%s is not an authorized user for this query.', self.user_id)
 
-        data = self.payload['data']
-        email_subject = data['email_subject']
-        email_body = data['email_body']
+        email_subject = self.payload['email_subject']
+        email_body = self.payload['email_body']
+        test_email_body = '[This is test email.]<br> %s' % email_body
```

Comment:
This is **a** test email. (I also suggest adding one more line braek to separate it from the main content.)